wikitransbot/main.py

```diff
@@ -1,5 +1,6 @@
 import json
 import logging
+import re
```

Comment:
nit: space missing

(I must add something like `black` to format in the CI/CD but I have not dived in it yet. :/)

Reply:
I just saw the Github formatting got me. No space was needed. Don't worry I'll fix it before merging. Apologies :pray: